config/locales/en/activerecord/course/assessment/question.yml

```diff
@@ -2,7 +2,7 @@ en:
   activerecord:
     course/assessment/question:
       question_number: 'Question %{index}'
-      question_with_title: '%{question_number}: %{title}'
+      question_with_title: '%{question_number}'
```

Comment:
Why is the title removed? Have you considered other consequences?

Reply:
understand that this `question_with_title` is used in lots of other places. My intention firstly was to split the `question_number` with `title`, but realising this, I have modified it and refactored it properly so that the SubmissionAnswer will still look as expected